misc/docker-compose.yml

```
@@ -7,7 +7,7 @@ services:
       dockerfile: ./misc/Dockerfile    
     environment:
       CONNECTION_STRING: "host=e2e-db port=5432 user=algorand password=algorand dbname=indexer_db sslmode=disable"
-      CI_E2E_FILENAME: "rel-nightly"
+      CI_E2E_FILENAME: "feature-avm-box"
```

```suggestion
      CI_E2E_FILENAME: "rel-nightly"
```

^^^ TODO: revert before merging ^^^